scripts/quarantine_zephyr.yaml

```
@@ -93,20 +93,35 @@
   comment: "Fixed with https://github.com/zephyrproject-rtos/zephyr/pull/68463"
 
 - scenarios:
-    - examples.nrfx_gppi.fork
-    - examples.nrfx_gppi.one_to_one
-    - examples.nrfx_saadc.simple_blocking
-    - examples.nrfx_saadc.simple_non_blocking
-    - examples.nrfx_saadc.advanced_blocking
-    - examples.nrfx_saadc.advanced_non_blocking_internal_timer
-    - examples.nrfx_saadc.maximum_performance
+    - examples.nrfx_uarte.tx_rx_non_blocking
   platforms:
-    - nrf52dk_nrf52832
-    - nrf52833dk_nrf52833
+    - nrf9160dk_nrf9160
     - nrf52840dk_nrf52840
+  comment: "Fixed with https://nordicsemi.atlassian.net/browse/NRFX-3395"
```

```suggestion
  comment: "https://nordicsemi.atlassian.net/browse/NRFX-3395"
```
Also in the other comments that specify the tracking issue, not the upstream PR that fixes the problem.

---

done